src/main/java/org/kuali/kfs/sys/document/service/impl/AccountingLineRuleHelperServiceImpl.java

```diff
@@ -0,0 +1,589 @@
+     *
+     * @param toCheck             the Object to check for existence
+     * @param attributeName       the name of the SourceAccountingLine attribute in the accountingLineEntry
+     * @param propertyName        the name of the property within the global error path.
+     * @return list of the validation errors based on existence of object
+     */
+    protected List<AccountingLineValidationError> getExistenceValidationErrors(Object toCheck, String attributeName, String propertyName,
+            String errorPropertyIdentifyingName, boolean useShortMessages) {
+        List<AccountingLineValidationError> validationErrors = new ArrayList<>();
+        String label = dataDictionaryService.getAttributeShortLabel(SourceAccountingLine.class, attributeName);
+        if (ObjectUtils.isNull(toCheck)) {
+            AccountingLineValidationError validationError = new AccountingLineValidationError(propertyName);
+            if (useShortMessages) {
+                validationError.setErrorKey(KFSKeyConstants.ERROR_EXISTING_SHORT);
+            } else {
+                validationError.setErrorKey(KFSKeyConstants.ERROR_EXISTING_WITH_IDENTIFYING_ACCOUNTING_LINE);
+                validationError.setMessageParameters(errorPropertyIdentifyingName, label);
+            }
+            validationErrors.add(validationError);
+        }
+        return validationErrors;
+    }
+
+    protected String getAccountIdentifyingPropertyName(AccountingLine accountingLine) {
+        String errorProperty = "";
+
+        // CORNELL FIX: if (accountingLine.getSequenceNumber() != null) {
+        if (ObjectUtils.isNotNull(accountingLine) && ObjectUtils.isNotNull(accountingLine.getSequenceNumber())) {
+            errorProperty = "Accounting Line: " + accountingLine.getSequenceNumber() + ", Chart: " +
+                    accountingLine.getChartOfAccountsCode() + ", Account: " + accountingLine.getAccountNumber() + " - ";
+        }
+
+        return errorProperty;
+    }
+
+    public void setBusinessObjectDictionaryService(
+            BusinessObjectDictionaryService businessObjectDictionaryService) {
+        this.businessObjectDictionaryService = businessObjectDictionaryService;
+    }
+
+    public void setDataDictionaryService(DataDictionaryService dataDictionaryService) {
+        this.dataDictionaryService = dataDictionaryService;
+    }
+
+    public void setFinancialSystemDocumentTypeService(
+            FinancialSystemDocumentTypeService financialSystemDocumentTypeService) {
+        this.financialSystemDocumentTypeService = financialSystemDocumentTypeService;
+    }
+
+    public void setAccountService(AccountService accountService) {
+        this.accountService = accountService;
+    }
+
+    public void setConfigurationService(ConfigurationService configurationService) {
+        this.configurationService = configurationService;
+    }
+
+    // This exists so tests can avoid static mocking
+    String getDashSubAccountNumber() {
```

Comment:
It looks like this overlay is for the 2020-12-17 version of the class. However, that release only seems to have made some minor updates for unit testing convenience, such as the addition of this particular method. I'm okay with keeping those changes in here if necessary.

Reply:
Removing these lines and adjusted two other area to keep file in sync with KualiCo 12-03-2020 released version.